src/tools/dart-compile.md

```diff
@@ -74,39 +75,43 @@ The following table shows the subcommands of `dart compile`.
   <tr>
     <td> <code>exe</code> </td>
     <td> <span style="white-space: nowrap">Self-contained</span> executable </td>
-    <td> A standalone, architecture-specific executable file.
-      <a href="#exe">Learn more.</a>
+    <td> A standalone, architecture-specific executable file containing the source code
+      compiled to machine code and a small <a href="/overview#runtime">Dart runtime</a>.
+      <br><em><a href="#exe">Learn more.</a></em>
     </td>
   </tr>
   <tr>
     <td style="white-space: nowrap"> <code>aot-snapshot</code> </td>
-    <td style="white-space: nowrap"> AOT snapshot </td>
-    <td> An architecture-specific file with <b>no Dart runtime</b>.
-      <a href="#aot-snapshot">Learn more.</a>
+    <td style="white-space: nowrap"> AOT module </td>
+    <td> An architecture-specific file, containing the source code
+      compiled to machine code, and <b>no Dart runtime</b>.
+      <br><em><a href="#aot-snapshot">Learn more.</a></em>
     </td>
   </tr>
   <tr>
     <td> <code>jit-snapshot</code> </td>
-    <td> JIT snapshot </td>
+    <td> JIT module </td>
```

Comment:
Are jit snapshots something we want to encourage people to use?  ( @mraleph @a-siva )

If so, when should they consider this over aot?

Reply:
I think we still have not settled on JIT vs AOT ourselves. As mentioned on the internal discussions: I do believe we might be leaving something like 30% performance on the table due team constraints which force us to have strong AOT focus.

People should consider JIT snapshots over AOT when they want peak performance and they have good training data.

Reply:
OK, added `      JIT-compiled code can have faster peak performance than AOT code
      in some cases, when the training data is good.
`